src/components/awfy/speedometer.js

```diff
@@ -54,14 +54,30 @@ export default class Speedometer extends Component {
 
   state = {
     series: undefined,
+    markers: undefined,
   }
 
   componentDidMount() {
-    this.fetchPlotGraph(this.props);
+    this.fetchMarkerData();
   }
 
   viewport: [0, 0];
 
+  transformMarkerData(data) {
```

Comment:
Couldn't you just do something like:

```js
return data.map(d => ({ date: d.date, label: d.version }));
```

(maybe even dispensing with the helper function.

Reply:
Thanks for the suggestion